packages/react-components/react-input/.storybook/preview.js

```diff
@@ -4,11 +4,4 @@ import * as rootPreview from '../../../../.storybook/preview';
 export const decorators = [...rootPreview.decorators];
 
 /** @type {typeof rootPreview.parameters} */
-export const parameters = {
```

Comment:
This is changing only in `react-input`, is it intentional?

Reply:
I have no context why there is used such a custom setup. Nothing was changed by hand from my side. If that's configuration has a reasoning I'd suggest to provide that reasoning for future migration runs, so person executing those automation has full context and thus can undo appropriately. migration generators are not using AST for storybook configs as that's not needed at the moment and to implement it the effort would be non trivial.

ty